Detectors/TOF/simulation/src/Strip.cxx

```diff
@@ -0,0 +1,107 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+//
+//  Strip.cxx: structure to store the TOF hits and digits in strips - useful
+// for clusterization purposes
+//  ALICEO2
+//
+#include <cstring>
+#include <tuple>
+
+#include <TMath.h>
+#include <TObjArray.h>
+
+#include "TOFSimulation/Strip.h"
+
+using namespace o2::tof;
+
+ClassImp(o2::tof::Strip);
+
+//_______________________________________________________________________
+Strip::Strip(Int_t index)
+  : mStripIndex(index)
+{
+}
+//_______________________________________________________________________
+Strip::Strip(const Strip& ref) = default;
```

Comment:
better to put this directly in header. Maybe we don't even need to mention the copy constructor?

Reply:
Moved to header.